.github/workflows/main.yml

```diff
@@ -1,5 +1,13 @@
 name: Build Extension
-on: [push, pull_request]
+on:
+  push:
+    paths:
+    - '**'
+    - '!**.md'
```

Comment:
This could be simplified to a single line with `paths-ignore`.

Reply:
Could, but I prefer this mechanism which explicitly gives the condition for running CI as "any of $these, but not $these" rather than a simple ignore --- for, empirically, I found that just doing `!**.extension` without a positive '**' before it had the consequence of CI *never* running, and even though `paths-ignore` probably does the right thing by itself, it's less obvious to me as a hypothetical reader-of-this-file-6-months-from-now that it does, if that makes sense?